.env.kesaseteli.example

```diff
@@ -18,5 +18,11 @@ EAUTHORIZATIONS_CLIENT_ID=
 EAUTHORIZATIONS_CLIENT_SECRET=
 EAUTHORIZATIONS_API_OAUTH_SECRET=
 
+ADFS_CLIENT_ID=
+ADFS_CLIENT_SECRET=
+ADFS_TENANT_ID=
+ADFS_LOGIN_REDIRECT_URL=
+ADFS_LOGIN_REDIRECT_URL_FAILURE=
```

Comment:
Where would these two redirect urls point in actual use?

Reply:
They should point to the handler UI success / error views. One thing I was thinking that should we create a separate django project for the handler as well, similarly as it is done in the frontend (at least for kesäseteli) but not sure if needed.

Reply:
ok! These URLs would not be used if suomi.fi authentication is used.

It's a good question, whether the handler and applicant code should be split... in Helsinki-benefit at least, I hope the same API (with appropriate permission checks) can be used dor both